wdl2cwl/main.py

```diff
@@ -331,6 +331,16 @@ def get_input(
                         ],
                     )
                 )
+            else:
+                inputs.append(
+                    cwl.CommandInputParameter(
+                        id=input_name,
+                        type=[
+                            cwl.CommandInputArraySchema(items=input_type, type="array")
+                        ],
+                        default="",
```

Comment:
`[]` would be a better default, as an empty string is doesn't match the type.

`defined` would also need to work differently for arrays, testing for length greater than zero instead of (in)equality with and empty string.